### DISCUSSION TITLE
Date Time field in 4.0 without default and differing behaviour

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#8118] [Troubleshooting] [answered]
Wouldn't it be nice to have both the date and time component of a date field to default to now()? 

I wonder how I can achieve this in the field settings of the blueprint but found no way.

I would also like that both components would act the same, so that clicking on the date and time icon would fill both elements with the current date or time. It's a bit weird that they behave differently, although I wouldn't mind if I could have a default value ...

**Comment by jasonvarga** [comment, marked as answer]
You can add `default: now` to the field.
